server/services/SlackBotService.ts

```diff
@@ -0,0 +1,49 @@
+import { Service } from "typedi";
+import { WebClient } from "@slack/web-api";
+import { SlackConfig } from "../Config";
+import { RoomsService } from "./RoomsService";
+import { RoomEvent } from "../express/types/RoomEvent";
+import { logger } from "../log";
+
+@Service({ multiple: false })
+export class SlackBotService {
```

Comment:
Why not just let typedi inject the config in a constructor here? Config is a bean anyways.